tests/travis/checkout_test_against_branch.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+SCRIPT_DIR=`dirname $0`
+
+if [ "$TEST_AGAINST_PIWIK_BRANCH" == "" ]; then
+    if [ "$TEST_AGAINST_CORE" == "latest_stable" ]; then # test against the latest stable release of piwik core (including betas & release candidates)
```

Comment:
I guess it means 'latest_release'. I guess we can make the git tag parsing more sophisticated to find a version w/o a -b[N] or -rc[N] etc. What do you think?